DEPS

```diff
@@ -942,6 +947,16 @@ deps = {
      'dep_type': 'cipd',
    },
 
+  'src/fuchsia/test_scripts': {
+     'packages': [
+       {
+        'package': 'chromium/fuchsia/test-scripts',
+        'version': Var('fuchsia_test_scripts_version'),
+       }
+     ],
+     'dep_type': 'cipd',
```

Comment:
Adding `'condition': 'host_os == "linux" and not download_fuchsia_sdk',` should fix the licensing issues

Reply:
Can we also add this below to the system images? This will prevent all the other CI runs from needing to download these resources, and should fix the last license issues.

Reply:
Done. I think it would be better to have `download_fuchsia_deps` which considers both `host_os` and `target_os`. But I am not sure if it's the practice in flutter.

Reply:
For the licensing issue, I am reading https://github.com/flutter/engine/tree/main/tools/licenses, the fuchsia images should not be checked as they are not text files.

Reply:
Updated ci/licenses_golden/excluded_files to ignore the images/ folder. I will monitor if the licensing check passes.